intl-config.js

```diff
@@ -1,5 +1,7 @@
-{
-  "supportedLocales": ["en-US", "de", "fr", "pt-BR", "es", "id"],
+require('habitat').load();
+
+module.exports = {
+  "supportedLocales": process.env.SUPPORTED_LOCALES || ["en-US", "de", "fr", "pt-BR", "es", "id"],
```

Comment:
I think it's reasonable to have this be a config for prod.

If you think about it, the only place where it's not going to be "*" is prod. So prod really is the only place that needs this to be configurable. Making me feel like this config is for prod.

It also makes it easier to update the list.